token-standard/splice-api-token-allocation-v2/openapi/allocation-v1.yaml

```diff
@@ -0,0 +1,202 @@
+# Copyright (c) 2025 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+openapi: 3.0.0
+info:
+  title: allocation off-ledger API
+  description: |
+    Implemented by token registries for the purpose of the use and management of
+    allocations by wallets and apps orchestrating the settlement of asset transfers.
+  version: 1.0.0
+paths:
+
+  /registry/allocations/v1/{allocationId}/choice-contexts/execute-transfer:
```

Comment:
TODO: convert to v2

Also consider whether we should/can share the OpenAPI definitions